index.js

```diff
@@ -151,6 +170,11 @@ function Hook (modules, options, onrequire) {
           filename = idWithoutPrefix
         }
       }
+    } else if (coreOnly) {
+      // `coreOnly` is `true` if this was a call to `process.getBuiltinModule`, in which case
+      // we don't want to return anything if the requested `id` isn't a core module
+      debug('call to process.getBuiltinModule with unknown built-in id - returning undefined')
+      return undefined
```

Comment:
Or perhaps call the original to be sure to get its behaviour. I realize it is returning undefined, and isn't likely to change. Calling the original could handle the same case as in the `} else {` block just below where there was double-monkey-patching of `require`.

```suggestion
      return self._origGetBuiltinModule.apply(this, args)
```

Reply:
Fixed in eb50f6d 👍